handlers/ec2/resources/handlers/ec2/handler.xml

```diff
@@ -147,10 +246,6 @@
                         <env key="EC2_INSTANCE_TYPE" value="${ec2.instance.type}" />
                         <env key="EC2_PUBLIC_NETWORK" value="${ec2.public.network}" />
                         <env key="EC2_MGMT_NETWORK" value="${ec2.management.network}" />
-                        <env key="EC2_SLICE_USER_EMAIL" value="${ec2.slice.user.email}" />

```

Comment:
Are these not needed anymore?

Reply:
No, not needed as they are being passed only for project creation in create.project macro. No longer needed for vm start